cli/lib/utils.ts

```diff
@@ -165,3 +170,43 @@ export function isValidClient(
 ): Client is Client & { client_id: string; name: string } {
   return !!Client.name && !!Client.client_id
 }
+
+/**
+ * Compare two scripts and print the diff between them
+ *
+ * Takes scriptType as a convenience parameter to enable nicer printing, but is
+ * otherwise agnostic as to what it's diffing.
+ */
+export function printScriptDiff<
+  T extends RuleDefinition | DBActionScriptDefinition
+>(diffs: [T, Change[]][], scriptType: 'db scripts' | 'rules' = 'rules') {
```

Comment:
`scriptType` looks wrong here. I think the first half of the union should be `db`. Also, I don't love the default to `rules`. Feels like you should always specify what you want it to do explicitly.

Reply:
I'll make it explicit, as above, but it's used for pretty printing, and thus the db name would be ambiguous.

Reply:
Ah, I can work around that though.